account.go

```diff
@@ -298,29 +312,32 @@ type AccountCapabilitiesTransfersParams struct {
 
 // AccountCapabilitiesParams represent allowed parameters to configure capabilities on an account.
 type AccountCapabilitiesParams struct {
-	ACSSDebitPayments       *AccountCapabilitiesACSSDebitPaymentsParams       `form:"acss_debit_payments"`
-	AUBECSDebitPayments     *AccountCapabilitiesAUBECSDebitPaymentsParams     `form:"au_becs_debit_payments"`
-	BACSDebitPayments       *AccountCapabilitiesBACSDebitPaymentsParams       `form:"bacs_debit_payments"`
-	BancontactPayments      *AccountCapabilitiesBancontactPaymentsParams      `form:"bancontact_payments"`
-	BoletoPayments          *AccountCapabilitiesBoletoPaymentsParams          `form:"boleto_payments"`
-	CardIssuing             *AccountCapabilitiesCardIssuingParams             `form:"card_issuing"`
-	CardPayments            *AccountCapabilitiesCardPaymentsParams            `form:"card_payments"`
-	CartesBancairesPayments *AccountCapabilitiesCartesBancairesPaymentsParams `form:"cartes_bancaires_payments"`
-	EPSPayments             *AccountCapabilitiesEPSPaymentsParams             `form:"eps_payments"`
-	FPXPayments             *AccountCapabilitiesFPXPaymentsParams             `form:"fpx_payments"`
-	GiropayPayments         *AccountCapabilitiesGiropayPaymentsParams         `form:"giropay_payments"`
-	GrabpayPayments         *AccountCapabilitiesGrabpayPaymentsParams         `form:"grabpay_payments"`
-	IdealPayments           *AccountCapabilitiesIdealPaymentsParams           `form:"ideal_payments"`
-	JCBPayments             *AccountCapabilitiesJCBPaymentsParams             `form:"jcb_payments"`
-	KlarnaPayments          *AccountCapabilitiesKlarnaPaymentsParams          `form:"klarna_payments"`
-	LegacyPayments          *AccountCapabilitiesLegacyPaymentsParams          `form:"legacy_payments"`
-	OXXOPayments            *AccountCapabilitiesOXXOPaymentsParams            `form:"oxxo_payments"`
-	P24Payments             *AccountCapabilitiesP24PaymentsParams             `form:"p24_payments"`
-	SEPADebitPayments       *AccountCapabilitiesSEPADebitPaymentsParams       `form:"sepa_debit_payments"`
-	SofortPayments          *AccountCapabilitiesSofortPaymentsParams          `form:"sofort_payments"`
-	TaxReportingUS1099K     *AccountCapabilitiesTaxReportingUS1099KParams     `form:"tax_reporting_us_1099_k"`
-	TaxReportingUS1099MISC  *AccountCapabilitiesTaxReportingUS1099MISCParams  `form:"tax_reporting_us_1099_misc"`
-	Transfers               *AccountCapabilitiesTransfersParams               `form:"transfers"`
+	ListParams               `form:"*"`
+	Account                  *string                                            `form:"-"` // Included in URL
```

Comment:
I don't think those two are correct. This is for the `capabilities` hash inside the Account creation, it's not a list API